pages/admin.tsx

```diff
@@ -173,13 +184,18 @@ export default function Admin() {
                             Add Membership
                         </Button>
                     </div>} */}
-                    {showAddUserForm && (
-                        <div className="flex justify-center">
-                            <UserFormCard onFormSubmit={createUser} onCancel={() => setShowAddUserForm(false)} />
-                        </div>
-                    )}
-                </div>
-            </section>
-        </>
-    );
+          {showAddUserForm && (
+            <div className="flex justify-center">
+              <UserFormCard
+                onFormSubmit={createUser}
+                onCancel={() => setShowAddUserForm(false)}
+              />
+            </div>
+          )}
+        </div>
+      </section>
+    </>
+  );
 }
+
+export default isAdmin(Admin);

```

Comment:
Should be authPage(Admin, ["admin"]) if you agree with my statements in my previous comments